okapi-core/src/main/java/org/folio/okapi/util/TcpPortWaiting.java

```diff
@@ -62,13 +63,12 @@ public void setMaxIterations(int maxIterations) {
   /**
    * Wait for process and server to be listening.
    * @param process Process to monitor
-   * @param startFuture async result
+   * @return startFuture async result
```

Comment:
The `@return` javadoc should be removed because it does not give any additional information. (At least `startFuture` should be removed).